DataTypes_Variables_Strings/DataTypes_Variables_Strings/obj/Debug/net6.0/DataTypes_Variables_Strings.csproj.FileListAbsolute.txt

```diff
@@ -0,0 +1,15 @@
+/Users/karinakrinickaa/Projects/IT-academy/HW_IT_Academy/DataTypes_Variables_Strings/DataTypes_Variables_Strings/bin/Debug/net6.0/DataTypes_Variables_Strings
+/Users/karinakrinickaa/Projects/IT-academy/HW_IT_Academy/DataTypes_Variables_Strings/DataTypes_Variables_Strings/bin/Debug/net6.0/DataTypes_Variables_Strings.deps.json
+/Users/karinakrinickaa/Projects/IT-academy/HW_IT_Academy/DataTypes_Variables_Strings/DataTypes_Variables_Strings/bin/Debug/net6.0/DataTypes_Variables_Strings.runtimeconfig.json
```

Comment:
please add .gitignore file :) there should be no .bin, .debug folders in repo :)